### ISSUE TITLE
Automotive Network Pentesting with scapy3k

### ISSUE THREAD

**Issue by polybassa** [phaethon/scapy#188]
Hi,
i've developed some additions to make scapy3k usable for automotive penetration testing.
Mainly I implemented sockets for CAN and ISO-TP and Packet definitions for ISO14229 and a properitary ISO13400 protocol.

I'm wondering, if there is an interest in the scapy3k community to merge these additions to mainline. 

Cheers

**Comment by phaethon** [comment]
Yes, absolutely. Can you prepare PR?

**Comment by polybassa** [comment]
Yes, I will prepare a PR. Can you give me some advice, where to place my code? 
I made my additions like a scapy AddOn. Therefore all my files lay around in the root directory of scapy at the moment.

**Comment by polybassa** [comment]
#189
